docs/operating-scylla/procedures/cluster-management/scale-up-cluster.rst

```diff
@@ -2,6 +2,12 @@
 Upscale a Cluster
 =================
 
+.. note::
+
+   Upscaling a cluster requires at least a quorum of nodes in a cluster to be available. 
```

Comment:
This is not true for adding core to a node

Reply:
It is true

Reply:
On second thought, it makes sense, as increasing the number of a node's core should follow with adding ranges.

Reply:
The reason is that shard count for each node is now distributed using the topology table which is updated through Raft. Updating it requires quorum, as any Raft command does.